ddprof-ffi/src/exporter.rs

```diff
@@ -275,8 +258,135 @@ pub unsafe extern "C" fn profile_exporter_send(
         Ok(HttpStatus(response.status().as_u16()))
     }() {
         Ok(code) => SendResult::HttpResponse(code),
-        Err(err) => SendResult::Failure(error_into_buffer(err)),
+        Err(err) => SendResult::Failure(err.into()),
+    }
+}
+
+#[export_name = "ddprof_ffi_SendResult_drop"]
+pub unsafe extern "C" fn send_result_drop(result: SendResult) {
+    std::mem::drop(result)
+}
```

Comment:
A not-so-small note on naming: we're introducing here a new suffix `_drop` which we hadn't used before. But we already have

* `_delete` => `ddprof_ffi_ProfileExporterV3_delete`
* `_dtor` => `ddprof_ffi_NewProfileExporterV3Result_dtor`
* `_free` => `ddprof_ffi_Profile_free`

Here's my suggestion: Let's use this PR to pick one suffix, and then rip off the band-aid (anyway we're breaking the API) and rename all functions for consistency.

I slightly favor aligning to the conventions of the calling language (calling it `_free`), but I don't particularly mind any of the other options -- since you're the one doing all the amazing work, I'll trust your judgement 😄